tests/integration/workflows/test_task_matching.py

```diff
@@ -0,0 +1,69 @@
+# -*- coding: utf-8 -*-
```

Comment:
You might want to rename the file to `test_tasks_matching.py` to match the module you are testing, or even replicate the dir structure and put it in `tests/integration/workflows/tasks/test_matching.py`